cypher-shell/src/main/java/org/neo4j/shell/ShellRunner.java

```diff
@@ -1,10 +1,6 @@
 package org.neo4j.shell;
 
-import org.neo4j.shell.cli.CliArgHelper;
-import org.neo4j.shell.cli.FileHistorian;
-import org.neo4j.shell.cli.InteractiveShellRunner;
-import org.neo4j.shell.cli.NonInteractiveShellRunner;
-import org.neo4j.shell.cli.StringShellRunner;
+import org.neo4j.shell.cli.*;
```

Comment:
I think in this specific example it reads better in the import statements. But I don't have strong opinions on this one way or the other. I can change my intellij settings.